Microsoft.Toolkit.Uwp.SampleApp/SamplePages/DispatcherHelper/DispatcherHelperPage.xaml.cs

```diff
@@ -19,7 +19,9 @@ private async void ExecuteFromDifferentThreadButton_Click(object sender, RoutedE
         {
```

Comment:
Should we update the samples.json file here to obsolete this and refer to the dispatcherqueue instead? Actually, is there no new DispatcherQueue Helper page? Should we just translate all these samples over to the new system?

Reply:
I updated the sample.